addons/web/static/src/views/list/list_renderer.js

```diff
@@ -383,7 +383,7 @@ export class ListRenderer extends Component {
             const fieldName = column.name;
             // in findNextFocusableOnRow test is done by using classList
             // refactor
-            if (!editedRecord.isReadonly(fieldName)) {
+            if (!editedRecord.isReadonly(fieldName) && this.tableRef.el) {
```

Comment:
tableRef.el should always exist, so I don't understand why this is necessary. Also, the commit message isn't really helpful for the js part

Reply:
@mobe-odoo I think you should revert js changes and focus on original error. The mentioned UX issue needs a review from PO before implementation. We can do it later.